index.test.js

```diff
@@ -43,26 +43,26 @@ describe('findByCode', function () {
 })
 
 describe('getHighestRiskRegion', function () {
-  it('should return the destinations region if only one destination', function () {
-    expect(libInsuranceCountries.getHighestRiskRegion(['NZ'])).to.equal('new zealand')
+  it('should return the destination if only one destination', function () {
+    expect(libInsuranceCountries.getHighestRiskRegion(['NZ'])).to.equal('NZ')
   })
 
-  it('should return the hightest risk region if multiple destinations', function () {
-    expect(libInsuranceCountries.getHighestRiskRegion(['NZ', 'JP'])).to.equal('asia')
+  it('should return the hightest risk destination if multiple destinations', function () {
+    expect(libInsuranceCountries.getHighestRiskRegion(['AU', 'NZ', 'JP'])).to.equal('JP')
   })
 
-  it('should return the worldwide region if multiple destinations', function () {
-    expect(libInsuranceCountries.getHighestRiskRegion(['AU', 'NZ', 'JP', 'EH'])).to.equal('worldwide')
-  })
-
-  it('should return domestic if travelling to in Australia', function () {
-    expect(libInsuranceCountries.getHighestRiskRegion(['AU'])).to.equal('domestic')
+  it('shouldnt return domestic country if any other passed in', function () {
+    expect(libInsuranceCountries.getHighestRiskRegion(['AU', 'NZ'])).to.equal('NZ')
```

Comment:
imo this is redundant but not a big deal

Reply:
Actually yeah you're right. Removing.